### ISSUE TITLE
Download links for external content need to redirect

### ISSUE THREAD

**Issue by little9** [psu-libraries/scholarsphere-3#1258]
With externally hosted content the content needs to be a redirect http status code not 200. Right now the`content-disposition` is correct, but browsers and `curl` or `wget` won't load the content from the external filename:
```
< HTTP/1.1 200 OK
< Date: Wed, 30 May 2018 01:58:56 GMT
< Connection: close
< X-Frame-Options: SAMEORIGIN
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< Accept-Ranges: bytes
< Content-Disposition: inline; filename="http://localhost:8000/repository/7p/v6/3f/z4/7pv63fz49w/Screenshot-2018-3-9_HathiTrust_Digital_Library_Millions_of_books_online.png"
< Content-Transfer-Encoding: binary
< Content-Type: image/png
< Content-Length: 0
< Last-Modified: Wed, 30 May 2018 01:11:28 GMT
< Cache-Control: private
< Set-Cookie: _scholarsphere_secure_session=QzVheDl3bmxiYTA0YitRZ1B0aEVqaXBUeDlQZTRUYjluQkZaeWNXNE43dzdEYU50Wm55eHV5aE9xZlpvdGdHd0hXUUpPN0MwUG01dzdvTHNERkJ5Nmc9PS0tV0VlZkxqNVVOdXl5dmg5SFhtRTgyQT09--3b20d901ab6296fdd950a0828255830ef866a11e; path=/; HttpOnly
< X-Request-Id: d022fda6-0020-41c4-b3b3-36dfa236770c
< X-Runtime: 0.178785
```

**Comment by little9** [comment]
From: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition
The filename shouldn't be an external url:

```
filename
Is followed by a string containing the original name of the file transmitted. The filename is always optional and must not be used blindly by the application: path information should be stripped, and conversion to the server file system rules should be done. This parameter provides mostly indicative information. When used in combination with Content-Disposition: attachment, it is used as the default filename for an eventual "Save As" dialog presented to the user.
```

**Comment by carolyncole** [comment]
@little9 I do not think we want to just forward the URL.  I think we may want to pull the content and pass it along like we do for content in Fedora.  That way we would not need security on the content HTTP server , that would all be handled by rails as it currently does.

**Comment by little9** [comment]
Fixed by #1271
